src/actions/tests/pending-breakpoints.js

```diff
@@ -0,0 +1,261 @@
+// TODO: we would like to mock this in the local tests
+const theMockedPendingBreakpoint = {
+  location: {
+    sourceUrl: "http://localhost:8000/examples/bar.js",
+    line: 5,
+    column: undefined
+  },
+  condition: "3",
+  disabled: false
+};
+
+jest.mock("../../utils/prefs", () => ({
+  prefs: {
+    expressions: [],
+    pendingBreakpoints: {
+      "http://localhost:8000/examples/bar.js:5:": {
+        location: {
+          sourceUrl: "http://localhost:8000/examples/bar.js",
+          line: 5,
+          column: undefined
+        },
+        condition: "3",
+        disabled: false
+      }
+    }
+  }
+}));
+
+import {
+  createStore,
+  selectors,
+  actions,
+  makeSource
+} from "../../utils/test-head";
+import {
+  makeLocationId,
+  makePendingLocationId
+} from "../../reducers/breakpoints";
+import expect from "expect.js";
+
+function generateBreakpoint(filename) {
+  return {
+    location: {
+      sourceUrl: `http://localhost:8000/examples/${filename}`,
+      sourceId: filename,
+      line: 5
+    },
+    condition: null,
+    disabled: false
+  };
+}
+
+function generatePendingBreakpoint(breakpoint) {
+  const {
+    location: { sourceUrl, line, column },
+    condition,
+    disabled
+  } = breakpoint;
+
+  return {
+    location: { sourceUrl, line, column },
+    condition,
+    disabled
+  };
+}
+
+function slideMockBp(bp) {
```

Comment:
same comment here as above on line 79 in breakpoints.js --- probably we can do this in a future pr, where we create a breakpoints utils